tempesta_fw/t/unit/sched_helper.c

```diff
@@ -133,3 +134,84 @@ test_conn_release_all(TfwSrvGroup *sg)
 		}
 	}
 }
+
+void test_sched_generic_empty_sg(struct TestSchedHelper * sched_helper)
```

Comment:
Please don't put a space between the asterisk `*` and the name of a variable. This applies to many places in this patch set.

Also, please put the function name on a separate line. This also applies to multiple places.